Makefile

```diff
@@ -58,13 +58,13 @@ include tasks/*.mk
 ###############################################################################
 
 ## Porcelain ##################################################################
-install: network-create vault-create assets-create app-create sandbox-create diversity-create proxy-create
+install: network-create vault-create assets-create app-create sandbox-create proxy-create
```

Comment:
If you remove the diversity tasks from the normal flow, are you going to provide another task, say `make diversity-install` to be able to deploy updates on that?